res/database/Default/lang/en.xml

```diff
@@ -174,10 +174,294 @@
+		<brand_vaio>VIAO</brand_vaio> <!-- https://en.wikipedia.org/wiki/Vaio -->
+		<brand_playstation>GameStation</brand_playstation> <!-- https://en.wikipedia.org/wiki/PlayStation -->
+		<brand_ps>GS</brand_ps>
+		<brand_nintendo>Nipponix</brand_nintendo> <!-- https://en.wikipedia.org/wiki/Nintendo | alt. spoof previously used: Nidennto -->
+		<brand_supernes>Super NIP</brand_supernes> <!-- https://en.wikipedia.org/wiki/Super_Nintendo_Entertainment_System -->
+		<brand_gamecube>NippoCube</brand_gamecube> <!-- https://en.wikipedia.org/wiki/GameCube -->
+		<brand_commodore>Cormoran</brand_commodore> <!-- https://en.wikipedia.org/wiki/Commodore_International -->
+		<brand_c64>Co64</brand_c64> <!-- https://en.wikipedia.org/wiki/Commodore_64 -->
+		<brand_datasette1530>Datasette 1350</brand_datasette1530> <!-- https://en.wikipedia.org/wiki/Commodore_Datasette -->
+		<brand_amiga>Novia</brand_amiga> <!-- https://en.wikipedia.org/wiki/Amiga | previous spoof: Eterna -->
+		<brand_cdtv>CDVID</brand_cdtv> <!-- https://en.wikipedia.org/wiki/CDTV -->
+		<brand_escom>ESCAL</brand_escom> <!-- https://en.wikipedia.org/wiki/Escom_AG -->
+		<brand_highscreen>HiScreen</brand_highscreen> <!-- https://de.wikipedia.org/wiki/Vobis -->
+		<brand_piranhabytes>Goldfish Bytes</brand_piranhabytes> <!-- https://en.wikipedia.org/wiki/Piranha_Bytes -->
+		<brand_rainbowarts>Rainbow Wish</brand_rainbowarts> <!-- https://en.wikipedia.org/wiki/Rainbow_Arts -->
+		<brand_rushware>Brushware</brand_rushware> <!-- https://de.wikipedia.org/wiki/Rushware -->
+		<brand_schneider>Schnitzler</brand_schneider> <!-- https://en.wikipedia.org/wiki/Schneider_Rundfunkwerke -->
+		<brand_atari>Arati</brand_atari> <!-- https://en.wikipedia.org/wiki/Atari -->
+		<brand_virgin>Virgo</brand_virgin> <!-- https://en.wikipedia.org/wiki/Virgin_Group -->
+		<brand_pioneer>Pyrodeer</brand_pioneer> <!-- https://en.wikipedia.org/wiki/Pioneer_Corporation -->
+		<brand_laserdisc>LazerDisk</brand_laserdisc> <!-- https://en.wikipedia.org/wiki/LaserDisc -->
+		<brand_siemens>Sumens</brand_siemens> <!-- https://en.wikipedia.org/wiki/Siemens | other extra-parody spoof: Hiemens -->
+		<brand_deutschetelekom>Deutsche Tellycom</brand_deutschetelekom> <!-- https://en.wikipedia.org/wiki/Deutsche_Telekom -->
+		<brand_telekom>Tellycom</brand_telekom>
+		<brand_bostondynamics>Bolton Dynamos</brand_bostondynamics> <!-- https://en.wikipedia.org/wiki/Boston_Dynamics -->
+		<brand_tesla>Tusla</brand_tesla> <!-- https://en.wikipedia.org/wiki/Tesla,_Inc. | TODO: Nikola Tesla not yet spoofed -->
+		<brand_mercedes>Mercieles</brand_mercedes> <!-- https://en.wikipedia.org/wiki/Mercedes-Benz -->
+		<brand_benz>Bonz</brand_benz> <!-- https://en.wikipedia.org/wiki/Mercedes-Benz -->
+		<brand_bmw>BWM</brand_bmw> <!-- https://en.wikipedia.org/wiki/BMW -->
+		<brand_volkswagen>Wolkwagen</brand_volkswagen> <!-- https://en.wikipedia.org/wiki/Volkswagen | alt./previous spoof: Faulkswagen; TODO: use for some parody -->
```

Comment:
Folkswaggon -> FW (maybe a bit too "literally translated" -> Folxwaggon ?
Worldvehicles -> WV

(avoid to call it Volksauto or something which could be translated to it - because the "Fuhrer" wanted a car following this term)

Reply:
I took out the "V." because it just looks dumb in text. It can be implied there is a "Verbund" or "Verband" or something like that.

Reply:
(I don't insist on a change here.)